src/main/proto/commit.proto

```diff
@@ -24,5 +24,7 @@ message Commit {
     string author_email = 4;
     string date = 5;
     bool qommit = 6;  // Quality commit.
-    repeated CommitStats commit_stats = 7;  // Stats on a commit.
+    int32 num_lines_add = 7;  // Number of lines for whole commit.
```

Comment:
Ok, will do it. I think that this fields necessary because if we can't recognize and calculate stats for all of the lines (unknown format for example) we will get the wrong aggregation.
I agree with you about rebase for squashing.